### ISSUE TITLE
PTX: Definitions

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/linear-algebra#163]
I had a LaTeX PDF containing only the definitions, I think it would be helpful to have this in the PTX version somehow

https://github.com/TeamBasedInquiryLearning/linear-algebra/blob/main/deprecated/pdf/definitions.pdf

**Comment by StevenClontz** [comment]
To keep things DRY we should implement this using PreTeXt features if possible.

Conversation: https://groups.google.com/g/pretext-support/c/HgvQM_k6SPg/m/tNqMrBsTBAAJ

**Comment by siwelwerd** [comment]
Digging into the sample article code from that email thread, I think this will work (have to run teach, will try it later)
```
<appendix xml:id="appendix-definitions">
      <title>Definitions</title>
      <list-of elements="definition" divisions="section subsection " empty="no" />
</appendix>

**Comment by StevenClontz** [comment]
Implemented in eae09f5a08f26e653bdb38ae68084fc29a7d20b2 and now available: https://teambasedinquirylearning.github.io/linear-algebra/list-of-definitions.html

Should be easy to add additional appendices for e.g. a list of activities, if there's a use-case.
